src/cpu/aarch64/jit_uni_eltwise.cpp

```diff
@@ -136,13 +155,30 @@ struct jit_uni_kernel_t : public jit_uni_eltwise_kernel {
         cmp(reg_work_amount, 0);
         b(LE, reminder_loop_end);
 
-        ld1(xmm_src[0], ptr(reg_src));
-        eltwise_injector_->compute_vector(xmm_src.getIdx());
+        if (is_bf16())
+            ld1(v_bf16[0], ptr(reg_src));
+        else
+            ld1(xmm_src[0], ptr(reg_src));
+
+        if (is_bf16()) {
+            // Convert BF16 input to FP32, apply eltwise op, then convert back to BF16
+            shll(VReg4S(v_bf16[0].getIdx()), VReg4H(v_bf16[0].getIdx()), 16);
```

Comment:
Can we avoid this getIdx and casting stuff by using a VReg directly and reinterpret with `.s` `.h` etc?